ethereum/clpps/CLPPSToken.sol

```diff
@@ -0,0 +1,113 @@
+pragma solidity >=0.4.22 <0.7.0;
+
+library SafeMath {
```

Comment:
i was thinking, since this is a private network, that we would need to manage/deploy every resource we intend to use.  but it would be cleaner to have this library written in a separate file and import it here 👍